src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,52 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {
+            tabs.map(tab => (
+              <li
+                key={tab.id}
+                className={
+                  classNames(null, {
+                    'is-active': tab.id === selectedTab.id,
+                  })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (selectedTab.id === tab.id) {
+                      return;
+                    }
+
+                    onTabSelected(tab);
+                  }
+                  }
```

Comment:
You could create a separate function for handling onClick, or you can use the `&&` operator to combine the condition and the function call in one line:
 ` ...=> condition && function() `